cosmoz-data-nav.js

```diff
@@ -435,6 +441,32 @@
 			}
 		}
 
+		_onItemsChange(change) {
```

Comment:
I think we should add one `_onItemsChange(items.*)` observer and treat a couple of cases:
- items is a new reference -> previous implementation
- items.splices -> this implementation
- items.*.prop -> forward to the instance.

Reply:
That's a great idea, thanks! Currently item prop updates are not forwarded. I'll add it to the spec.

Reply:
Isn't prop forwarded to the instance via the templatizing stuff?

Reply:
@nomego Templatizing only forwards host (data-nav) props, not item props.
To forward item props, one would have to implement an items.* observer that forwards paths (e.g. items.2.address).

Reply:
Ok but the template instance itself bind to the object passed as "item", so any changes to the object should be reflected in the view anyway?
Even if those changes occur through a reference from list-queue or wherever?

Reply:
@nomego Actually it does not work that way. Templatized templates have no way of knowing that item data has changed. It is the job of the host element to observe data changes and forward relevant paths to the correct instances.
I have added a test to the spec to detail this behavior. https://github.com/Neovici/cosmoz-data-nav/blob/0e5118e6e08a6f2f69d908b16998c79593dfec42/test/spec.html#L156
It can be done, but it would require a sizable effort to refactor and improve the current implementation.

Reply:
Ok well we can skip that for now. Need to have a look at that stuff when I get time :)